go.mod

```diff
@@ -16,11 +16,12 @@ require (
 	github.com/golang/mock v1.3.1
 	github.com/golang/protobuf v1.3.1
 	github.com/google/pprof v0.0.0-20190515194954-54271f7e092f // indirect
+	github.com/iotexproject/go-p2p v0.2.10
```

Comment:
Curious why election now needs to depend on P2P library. Can you help to investigate it too?

Reply:
sure